materials/section-1/src/koans/01_equalities.clj

```diff
@@ -2,14 +2,15 @@
   (:require [koan-engine.core :refer :all]))
```

Comment:
One general thing to note here - you'll have to be quite strict on time if you want everyone to make enough progress (even if you allocate something like 2 hours for this, you'll want to be finishing a file every ~7 mins).

It might be worth letting people know that they shouldn't be afraid to skip to the next file if they get stuck on the current one, just so they can keep a bit closer on time

Reply:
Also if people use a proper editor like emacs (or even VScode, maybe?) they wouldn't run into these indentation issues but perhaps that's a crusade for another day 😄

Reply:
This is true! There has to be a way to mark a koan as optional without commenting it out. Have to check it out. Otherwise I will leave it for the time being.